source/source-connector/usage-examples/topic-naming.txt

```diff
@@ -117,10 +127,117 @@ mapping, the connector performs the following actions:
+   namespace as a regex pattern or raises a ``ConnectConfigException``
+   if the ``/`` character is not at the beginning of the namespace.
+
+The connector computes the topic name by doing variable expansion on the
+topic name template. The connector supports the following variables: 
+ 
+- ``db``: The database name from the matching namespace.
+- ``sep``: The value of the ``topic.separator`` configuration property.
+  To learn more about this property, see :ref:`source-configuration-kafka-topic`.
+- ``coll``: The collection name from the matching namespace, or an empty
+  string if there is no collection name.
+- ``sep_coll``: The value of the ``coll`` variable prefixed with the value
+  of the ``sep`` variable if the value of ``coll`` is not empty.
+- ``coll_sep``: The value of the ``coll`` variable suffixed with the
+  value of the ``sep`` variable if the value of ``coll`` is not empty.
+- ``sep_coll_sep``: The value of the ``coll`` variable prefixed and
+  suffixed with the value of the ``sep`` variable if the value of
+  ``coll`` is not empty.
+ 
+You must enclose variables between curly brackets (``{}``) for the connector to
+expand them. You can't use curly brackets in the topic name template for
+any other purpose.
+
+.. note:: Escaping Characters
+
+   You must follow the JSON syntax to create a namespace pattern that includes
+   characters that need escaping. For example, to match
+   ``'.'``, the regex syntax requires that you escape it as ``'\.'``
+   You must escape the reverse solidus ``'\'`` character as ``'\\'``.
+   Consequently, to match ``'.'`` in your namespace pattern, you must
+   write it as ``'\\.'``.
+ 
+The connector matches namespaces by using the following order:
+ 
+1. Pairs with database and collection names in the namespace pattern. To
+   learn more about this namespace pattern, see the :ref:`Topic
+   Namespace Map Example <topic-naming-namespace-map-example>`.
+#. Pairs with only a database name in the namespace pattern. To
+   learn more about this namespace pattern, see the :ref:`Topic
+   Namespace Map Example <topic-naming-namespace-map-example>`.
+#. Regex pairs in order.
+#. The wildcard pair. To learn more about this namespace pattern, see
+   the :ref:`Topic Namespace Map with Wildcard Example
+   <topic-naming-namespace-map-wildcard-example>`.
+
+This example shows how to specify the ``topic.namespace.map``
+setting to define the following topic namespace mapping:
+
+- Maps a regular expression to the topic ``industrial{sep_coll}``. The
+  regex pattern matches any database names that include the string
+  ``'vertical'``, even if the database name has other characters.
+
+- Maps the ``vertical.health`` namespace to the ``healthcare`` topic.
+
+.. code-block:: ini
+
+   topic.namespace.map={"/vertical(?:\\..*)?": "industrial{sep_coll}", "vertical.health": "healthcare"}
+
+Since the ``vertical.health`` namespace mapping takes precedence over
```

Comment:
I: [Since](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#s:~:text=press%20Enter.-,since,-Use%20since%20to)

```suggestion
Because the ``vertical.health`` namespace mapping takes precedence over
```